spec/models/recipe_spec.rb

```diff
@@ -12,4 +12,15 @@
     it {should have_many :recipe_ingredients}
     it {should have_many(:ingredients).through(:recipe_ingredients)}
   end
+
+  before(:each) do
+    @recipe_1 = Recipe.create!(name: "Rice", complexity: 1, genre: "Rice Genre")
+    @ingredient_1 = @recipe_1.ingredients.create!(name: "Beef", cost: 40)
+    @ingredient_2 = @recipe_1.ingredients.create!(name: "Chicken", cost: 20)
+  end
+  describe "#total_cost" do
+    it "give the total cost" do
+      expect(@recipe_1.total_cost).to eq(60)
```

Comment:
Good model test example. For model tests, I try to include multiple cases... meaning: 
1. a normal case where it will return a result (you're doing this already)
2. a similar case where it just returns a _different_ result
3. a "negative" case where it returns 0 (or false, or `[]`, etc.) 

The reason for this is that we want to ensure a test doesn't just _always_ return `60`, for example. So, we make another recipe with 3-4 ingredients, and another recipe with 0 ingredients which should return `0` for its `total_cost`.